### DISCUSSION TITLE
Unable to redeem credit for Microsoft for startup

### DISCUSSION THREAD

**Discussion by islammds** [community/community#77820] [Other Feature Feedback, Questions, & Ideas]
### Hi everyone,

I forgot to copy the promo code from Microsoft for Startup portal and now I can not redeem the credit and cannot add my employees. I opened another ticket for this issue but got no response. Please help and it is urgent. Thanks
Md

**Comment by dakshithadissanayaka** [comment]
I understand that you're unable to redeem credit for Microsoft for Startup and that you haven't received a response to your support ticket. Here are a few steps you can take to resolve this issue:

1. **Check your inbox:** Double-check your inbox, including spam and promotions folders, to see if you can find the promo code from Microsoft for Startup. It's possible you may have missed or overlooked the email containing the code.

2. **Contact Microsoft for Startup support:** Reach out to Microsoft for Startup support directly to inquire about retrieving the lost promo code. They may be able to provide you with the code again or offer alternative solutions.

3. **Open a new support ticket:** If you haven't received a response to your previous support ticket, open a new one and clearly explain the situation. Provide any relevant details, such as the date you opened the previous ticket and the reference number if available.

4. **Check eligibility criteria:** Ensure your organization meets the eligibility criteria for Microsoft for Startup. Review the eligibility requirements and confirm that your startup aligns with the program's guidelines.

5. **Seek community support:** Engage with the Microsoft for Startup community on forums or social media platforms to see if others have encountered similar issues and if they have found solutions.

6. **Alternative verification:** Explore alternative verification methods offered by Microsoft for Startup, if any, that don't rely on the promo code. This might involve providing additional documentation or undergoing a manual verification process.

Remember to be patient and persistent in your efforts to resolve this issue. Microsoft for Startup support will be able to guide you further if any additional steps are required.

**Comment by samus-aran** [comment]
Hi @islammds ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
